build.sbt

```diff
@@ -71,6 +70,7 @@ lazy val microservice = Project(appName, file("."))
   .settings(publishingSettings: _*)
```

Comment:
Can you address the deprecation warning to remove `publishingSettings`

Reply:
removed ,